Procfile

```diff
@@ -1 +1 @@
-handlers: node lib/main.js server
```

Comment:
@jhford has suggested changing the best practice for this to `lib` since this is no longer source code that is fed to a compiler -- ec2-manager is leading the way with this new BP.  So we should probably do the same here, then fix the rest of the services up in the next round of spring cleaning?

Reply:
@jhford: can we keep these in `src` at least until the time when our libraries stop compiling too? I don't want to have to remember that I'm supposed to do things in `src` vs `lib` when I go to a new project.